os/manual-rollbacks.md

```diff
@@ -1,12 +1,10 @@
 # Performing manual CoreOS Container Linux rollbacks
 
-As much as we all love flawless, automatic updates, there may be occasions when an update must be rolled back. This is fairly straightforward on Container Linux, once you know the right commands.
+In the event of an upgrade failure, Container Linux will automatically boot with the version on the rollback partition. Immediately after an upgrade reboot, the active version of Container Linux can be rolled back to the version installed on the rollback partition, or downgraded to the version current on any lower release channel. 
```

Comment:
Explicit sentence to conclude this graf: "There is no method to downgrade to an arbitrary version number." (ref original intent of https://github.com/coreos/docs/issues/964)